packages/webview_flutter/tizen/project_def.prop

```diff
@@ -16,12 +16,12 @@ USER_CPP_UNDEFS =
 
 # Compiler/linker flags
 USER_CFLAGS_MISC =
-USER_CPPFLAGS_MISC = -c -fmessage-length=0
-USER_LFLAGS = -llightweight-web-engine
+USER_CPPFLAGS_MISC = -c -fmessage-length=0 
+USER_LFLAGS = -llightweight-web-engine.mobile -Wl,-rpath=/opt/usr/globalapps/org.tizen.webview_flutter_tizen_example/lib/arm
```

Comment:
I think that this path is for example app only

Reply:
As you said, that's right. This is just temporary code for convenience and we need an extensive discussion of the lwe distribution method.

Reply:
ok, thank you for explanation